app/assets/stylesheets/application.css.scss

```diff
@@ -168,3 +168,11 @@ div#bp_ad_end {
  .disable{
      opacity: 0.5
  }
+
+ meter {
+   width: 100%;
+   height: 40px;
+   border: 1px solid #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.